scout/server/blueprints/variants/templates/variants/fusion-variants.html

```diff
@@ -59,7 +59,8 @@
             <th style="width:3%"></th>
             <th style="width:4%" title="Beta version. Currently testing">Rank (beta)</th>
             <th style="width:6%" title="Genes">Fusion Genes</th>
-            <th style="width:4%" title="Number of callers detecting a given event">Callers</th>
```

Comment:
Hm, I would leave it at "Callers", thinking that even if it is a count currently, it will likely be either badges or a hover with the names of the callers in the not too distant future..